docs/beamrunner/beamrunner-4-monitor.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: docs
+title:  Monitoring IBM® Streams Runner for Apache Beam

```

Comment:
Is "Monitoring your running application" a more correct title? This sounds like we're monitoring the runner itself.